gulpfile.babel.js

```diff
@@ -243,56 +186,29 @@ gulp.task('scripts', ['lint'], () => {
       sourceMapIncludeSources: true
     }))
     .pipe($.header(banner, {pkg}))
-    .pipe($.concat('material.min.js'))
+    .pipe($.rename({extname: '.min.js'}))
     // Write Source Maps
     .pipe($.sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'))
-    .pipe($.size({title: 'scripts'}));
-});
-
-// Clean Output Directory
-gulp.task('clean', () => del(['dist', '.publish']));
```

Comment:
We don’t have that. Everything is in `dist`. A clean will delete `dist`. But I kinda see the point that a developer might expect that task to exist because it is so common. Will fix.